devito/ir/iet/scheduler.py

```diff
@@ -80,13 +81,15 @@ def _lower_stepping_dims(iet):
         # two different calls to `xreplace`
         mindices = [d for d in i.uindices if d.is_Modulo]
         groups = as_mapper(mindices, lambda d: d.modulo)
+        i_org = i
         for k, v in groups.items():
             mapper = {d.origin: d for d in v}
             rule = lambda i: i.function.is_TimeFunction and i.function._time_size == k
```

Comment:
Unrelated to this PR, but since you're editing this code.... instead of `lambda i` could you name it differently , since `i` is already used as loop iteration variable? Same for the line below

Reply:
Sure!